shoop/utils/multilanguage_model_form.py

```diff
@@ -87,6 +87,12 @@ def __init__(self, **kwargs):
         kwargs["initial"] = object_data
         super(MultiLanguageModelForm, self).__init__(**kwargs)
 
+        if all([self.instance, self.instance.pk, hasattr(self.instance, "protected_fields")]):
```

Comment:
I'm not sure this is very good use of `all` :) Also, that oughta be `getattr` with an empty tuple as the default arg to be clear...